### PR TITLE
feat(shop-list.html): use ul element to show the items

### PR DESCRIPTION
**Pull request by felixzapata**
Hi, due to the shop list is a list of items, I propose to use the `ul` element to markup the items.

Regards.

### CODE REVIEW

**Comment by blasten** [comment]
I'm not sure what the benefit would be for this particular case..

**Comment by felixzapata** [comment]
hi, mainly two:

- **semantic**: you are showing a list of items so you should use the element for that. In this case, the `ul` element. The same way you use a `p` element to represent a paragraph or a `label` element to represent a form label instead of something like this `<div class="label">your name</div>`.
- **accessibility**: when a screen reader find an `ul` element, it informs the user how many elements are inside on it.

Regards

File: src/shop-list.html [hunk 1]
```diff
@@ -98,11 +107,13 @@
     </header>
 
     <div class="grid" hidden$="[[failure]]">
+      <ul>
```
Comment (blasten) [inline src/shop-list.html]: maybe make this element `<ul class="grid" hidden$="[[failure]]">` and remove the outer div element.

**Comment by felixzapata** [comment]
@blasten done

File: src/shop-list.html [hunk 2]
```diff
@@ -97,13 +103,15 @@
       <span>[[_getPluralizedQuantity(category.items.length)]]</span>
     </header>
 
-    <div class="grid" hidden$="[[failure]]">
+
```
Comment (blasten) [inline src/shop-list.html]: this new line isn't needed

File: src/shop-list.html [hunk 3]
```diff
@@ -97,13 +103,15 @@
       <span>[[_getPluralizedQuantity(category.items.length)]]</span>
     </header>
 
-    <div class="grid" hidden$="[[failure]]">
+    
+    <ul class="grid" hidden$="[[failure]]">
       <template is="dom-repeat" items="[[_getListItems(category.items)]]" initial-count="4">
-        <a href$="[[_getItemHref(item)]]">
+        <li><a href$="[[_getItemHref(item)]]">
```
Comment (blasten) [inline src/shop-list.html]: Indentation: 
```html
<li>
  <a>
```

**Comment by blasten** [comment]
minor comments, otherwise LGTM

**Comment by felixzapata** [comment]
@blasten done

**Comment by blasten** [comment]
Thanks!

**Pull request merged**